nemo/collections/asr/models/ctc_models.py

```diff
@@ -714,3 +714,18 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
 
         temporary_datalayer = self._setup_dataloader_from_config(config=DictConfig(dl_config))
         return temporary_datalayer
+
+    def load_state_dict(self, state_dict, strict=True):
```

Comment:
@redoctopus let's revert this in 1.9 ?

Reply:
Yep, that's the plan. We're deprecating the model that relies on this hack in 1.9.